bugbug/models/component.py

```diff
@@ -132,6 +132,8 @@ def __init__(self, lemmatization=False):
 
     def filter_component(self, product, component):
         full_comp = f"{product}::{component}"
+        if product == "Fenix":
+            return "Fenix"
```

Comment:
I would move this to the beginning of the method (i.e., before `full_comp`).

Reply:
Done in 896d05dd6a91069c918c2a51f55d9292c6669344!

Reply:
Will generate metrics on the original model before this PR ASAP.